lib/src/__tests__/DatePickerProps.test.tsx

```diff
@@ -63,15 +63,15 @@ describe('DatePicker - different props', () => {
     expect(component.find('h4[data-mui-test="datepicker-toolbar-date"]').text()).toBe('January');
   });
 
-  it('showTodayLabel – accept current date when "today" button is clicked', () => {
+  it('showTodayButton – accept current date when "today" button is clicked', () => {
     const onCloseMock = jest.fn();
     const onChangeMock = jest.fn();
     const component = mount(
       <MobileDatePicker
         renderInput={props => <TextField {...props} />}
         autoOk
         showTodayButton
-        cancelLabel="stream"
+        cancelText="stream"
```

Comment:
Any reason for providing a string in this test?

Reply:
It is probably not the best way, but I am testing typescript bindings for props such way. 
Probably need to add some special kind of typescript definition tests

Reply:
So the purpose is to have a test equivalent to, say, https://github.com/mui-org/material-ui/blob/a75c19f267edfdf71487764961de5c425096f47b/packages/material-ui/src/Typography/typography.spec.tsx?